crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -1592,6 +1635,7 @@ pub enum TomlLintLevel {
 }
 
 #[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "unstable-schema", derive(schemars::JsonSchema))]
 pub struct InvalidCargoFeatures {}
```

Comment:
This should be skipped rather than be in the xhema

Reply:
deleted this line as invalidcargofeatures is skipped anyway